internal/store/migrations.go

```diff
@@ -2214,6 +2214,29 @@ var migrations = []migration{
 			return err
 		}
 
+		return nil
+	}},
+	{semver.MustParse("0.47.0"), semver.MustParse("0.48.0"), func(e execer) error {
+		_, err := e.Exec(`ALTER TABLE Cluster ADD COLUMN SchedulingLockAcquiredBy TEXT NULL;`)
+		if err != nil {
+			return err
+		}
+
+		_, err = e.Exec(`ALTER TABLE Cluster ADD COLUMN SchedulingLockAcquiredAt BIGINT;`)
+		if err != nil {
+			return err
+		}
+
+		_, err = e.Exec(`UPDATE Cluster SET SchedulingLockAcquiredAt = 0;`)
```

Comment:
Hey, may not fully understand your proposal, but I was avoiding setting a default here so that you have to explicitly inject the value when creating a new row. This matches the existing `LockAcquiredAt` logic so wanted to keep it the same here.

Reply:
Exactly the sort of reason I was looking for :)